nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/files/impl/FileStorageManager.java

```diff
@@ -119,5 +125,35 @@ private boolean createDirectory(String dirPath) {
     public void setFileTypeFolder(String fileTypeFolder) {
         this.fileTypeFolder = fileTypeFolder;
     }
-
+    
+    public String getStoragePath(String username, String uploadId,
+            String documentId) {
+        return baseDirectory + File.separator 
+                + getFileFolderPathInBaseFolder(username, uploadId, documentId);
+    }
+    
+    public void deleteFile(String username, String uploadId, String documentId, String fileName, boolean deleteEmptyFolders) throws NepomukFileStorageException {
```

Comment:
it looks like this method called is only once with `deleteEmptyFolders` set to true. Is that flag needed?